packages/replay/src/eventBuffer/EventBufferArray.ts

```diff
@@ -30,6 +33,12 @@ export class EventBufferArray implements EventBuffer {
 
   /** @inheritdoc */
   public async addEvent(event: RecordingEvent): Promise<AddEventResult> {
+    const eventSize = JSON.stringify(event).length;
```

Comment:
This seems expensive to be running on each event. This is running before we pass the event to the web worker?
Do we stringify this in the transport? Or elsewhere so we can measure at that point?

Reply:
Yes, in the non-worker based event buffer it adds overhead as we need to stringify as we go. In the buffer-based worker we already do that, so there it adds no overhead. If we want to keep track in any way, we will have to add overhead to the non-worker based buffer (currently, we only stringify on `finish()`, which is the most performant - but gives us no idea how large we are as we go).